uo_sys_framework/AILinkModule/cfgFunctions/fn_EvalRadio.sqf

```diff
@@ -1,9 +1,13 @@
+#define COMPONENT AILink
+#include "\x\UO_FW\addons\main\script_macros.hpp"
+UO_FW_EXEC_CHECK(ALL);
+
 private ["_groups","_thisGroup","_thatGroup","_thisLR","_thisSR","_thatLR","_thatSR","_range"];
 
 if (isNil "tin_aiLink_debug") then {tin_aiLink_debug = false};
 if (tin_aiLink_transDelay > tin_aiLink_shareDelay) then {tin_aiLink_transDelay = tin_aiLink_shareDelay};
 
-_allLinkGroups = [];
+private _allLinkGroups = [];
```

Comment:
This function is nearly equal to fn_AILink. Maybe it will be worth making a function with parameters that is used by both functions fn_AILink and fn_evalRadio